server/py_files/Spacy.py

```diff
@@ -5,18 +5,24 @@
 nlp = en_core_web_sm.load()
 nlp.max_length = 2000000 # todo check this limit
 
-import time
-
 class Spacy(object):
-    def anonymize(self, t):
-        if t.like_email: # todo bring in Human Names as well
-            return 'X'*len(t.text)
+    def anonymize_token(self, t):
+        if t.like_email:
+            return t.shape_
+
+        # todo we need a better way - this is also hiding some tools names which we need in our analysis (eg: Docker, Jenkins etc)
```

Comment:
@tzunun : commenting out this - I tested it and its creating some side effects as mentioned in the example - we might need a more nuanced solution than this (putting in backlog, don't think its worth the effort right now)